projects/WeTek_Play/options

```diff
@@ -111,13 +111,13 @@
     KODIPLAYER_DRIVER="libamcodec"
 
   # Modules to install in initramfs for early boot
-    INITRAMFS_MODULES="softcursor bitblit font fbcon"
+    INITRAMFS_MODULES=""
 
   # additional drivers to install:
   # for a list of additinoal drivers see packages/linux-drivers
   # Space separated list is supported,
   # e.g. ADDITIONAL_DRIVERS="DRIVER1 DRIVER2"
-    ADDITIONAL_DRIVERS=""
+    ADDITIONAL_DRIVERS="RTL8192CU RTL8192DU RTL8188EU"
```

Comment:
these 3rdparty drivers are horrible. see what happens on the rpi front.. sure, can be added if you want them so much, but later. lets fix the remaining issues first before adding stuff we dont/cant support at all. my 2 cents.